src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings

```diff
@@ -507,6 +514,8 @@
 
 "Home_Risk_No_Date_Title" = "Unknown";
 
+"Home_Risk_Period_Update_Title" = "Updated every %@ hours";
```

Comment:
non-breakable-space? same for German file

Reply:
done